WordPress/UITests/Tests/NotificationTests.swift

```diff
@@ -13,26 +14,38 @@ private extension String {
 
 class NotificationTests: XCTestCase {
     override func setUpWithError() throws {
-        try super.setUpWithError()
         setUpTestSuite()
+        WireMock.resetScenario(scenario: "comment_flow")
+        WireMock.fetchScenarios { (data, error) in
+            guard error == nil else {
+                print("Error fetching scenarios: \(error!)")
+                return
+            }
+        }
```

Comment:
Since `reset` and `fetch` will always be called together, what do you think about encapsulating them and calling something like `setUp` or any other name you find more appropriate? 

```suggestion
        WireMock.setUpScenario("comment_flow")
```

In the future we could pass an array of Strings in case we need multiple scenarios.

Reply:
Yeah I think this makes sense, I'm trying to think if there's a case when we need to call them separately but realized that we can still do that if needed (for now, i've made the two `private` since they are only called by the new method). 

Made the change to encapsulate them in https://github.com/wordpress-mobile/WordPress-iOS/pull/21288/commits/76a21942a94e03ff448381c2ce65d7a44cc4f471 (however, currently still only accepts one scenario, the change to make it an array will be done separately in a future PR)